Library/Formula/postgresql.rb

```diff
@@ -77,8 +82,22 @@ def install
   end
 
   def post_install
-    unless File.exist? "#{var}/postgres"
-      system "#{bin}/initdb", "#{var}/postgres"
+    pg_version = (var/"postgres/PG_VERSION").read.chomp
+
+    if (var/"postgres").exist? && (var/"postgres-#{pg_version}").exist?
+      puts "We have not attempted to migrate your data due to your multi-postgresql setup."
+    end
```

Comment:
This particular block was born out of Tim's concerns in [this comment](https://github.com/Homebrew/homebrew/pull/35577#discussion_r22504626).

If there's a better way to handle Tim's concerns around people darting between versions, count me in :).